components/avatar.css

```diff
@@ -0,0 +1,62 @@
+.container{
+    display: flex;
+    justify-content: space-evenly;
+    margin: 2rem;
+}
```

Comment:
As I can see you are using this to contain the avatar component but this is not the part component avatar so this line of code should be in a different file.